twitter-langchain/examples/chatbot/chatbot.ts

```diff
@@ -0,0 +1,215 @@
+import { TwitterToolkit } from "@coinbase/twitter-langchain";
+import { TwitterAgentkit } from "@coinbase/cdp-agentkit-core";
+import { HumanMessage } from "@langchain/core/messages";
+import { MemorySaver } from "@langchain/langgraph";
+import { createReactAgent } from "@langchain/langgraph/prebuilt";
+import { ChatOpenAI } from "@langchain/openai";
+import * as dotenv from "dotenv";
+import * as readline from "readline";
+
+dotenv.config();
+
+const modifier = `
+  You are a helpful agent that can interact with the Twitter (X) API using the Coinbase Developer Platform Twitter (X) Agentkit.
+  You are empowered to interact with Twitter (X) using your tools.
+
+  If someone asks you to do something you can't do with your currently available tools, you must say so, and encourage them to implement it themselves using the Twitter (X) API + Agentkit.
+  Recommend they go to https://developer.x.com/en/docs for more informaton.
+
+  Be concise and helpful with your responses.
+  Refrain from restating your tools' descriptions unless it is explicitly requested.
+`;
+
+/**
+ * Initialize the agent with Twitter (X) Agentkit
+ *
+ * @returns Agent executor and config
+ */
+async function initialize() {
+  // Initialize LLM
+  const llm = new ChatOpenAI({ model: "gpt-4o-mini" });
+
+  // Twitter (X) Agentkit options - these are loaded automatically via the ENV
+  const options = {
+    apiKey: "$TWITTER_API_KEY",
+    apiSecret: "$TWITTER_API_SECRET",
+    accessToken: "$TWITTER_ACCESS_TOKEN",
+    accessTokenSecret: "$TWITTER_ACCESS_TOKEN_SECRET",
```

Comment:
Bearer token is also required?

Reply:
<c/p from the prev comment>

this is an oddity that i dug into. there is a discrepancy between the libs being used in agentkit.py and agentkit.js. the python twitter library will not get account mentions without the bearer token provided however, the nodejs lib is happy with just the app and token keys and secrets. 

the bearer token is used for twitter-api-v2 integration. i think the nodejs libary is either calling the v1 mentions, or is obtaining a v2 token behind the scenes.

was thinking that maybe we could try another twitter library for agentkit.py, or maybe patch twit to remove the bearer token dep?